### DISCUSSION TITLE
Push translate and run dev

### DISCUSSION THREAD

**Discussion by NikolaKraus** [AndresMorelos/InvoncifyReleases#116] [Q&A] [answered]
Hi,
I'm sorry for stupid question but I'm new here. Can somebody tell me easy instructions or link to instructions about how to run clone of Invoncify (step by step would be best for me)? I was looking for it but every time It writes me errors. I tried to install electron lib to the project by comands like "npm install --save-dev electron-rebuild --force" and of course it installed node_modules but when I try to run projec it writes
"(node:9288) electron: Failed to load URL: file:///D:/O/Programy/Invoncify/entrypoint/tour.html with error: ERR_FILE_NOT_FOUND
(Use `electron --trace-warnings ...` to show where the warning was created)
(node:9288) electron: Failed to load URL: file:///D:/O/Programy/Invoncify/entrypoint/main.html with error: ERR_FILE_NOT_FOUND
(node:9288) electron: Failed to load URL: file:///D:/O/Programy/Invoncify/entrypoint/preview.html with error: ERR_FILE_NOT_FOUND"

Next question is how can I push CZ json files translated from SK? When I try to push it, GIT writes me that I do not have permission.

Thanks for response.
Regards
Nik

**Comment by AndresMorelos** [comment, marked as answer]
@NikolaKraus here are the instructions:

1. Create a fork of the repo (https://docs.github.com/en/get-started/quickstart/fork-a-repo)
2. Clone your fork on your machine.
3. Open the folder in a terminal and install the dependencies (ensure that you're using yarn) `yarn install`

Now you need two terminals because to execute the app you'll have to run two commands:

[On the first terminal]
4. Execute the command `yarn dev`
[On the second terminal]
5. Execute the command once the first command has finish`yarn start:dev`

----------

To add a new language please visit the crowding page https://crowdin.com/project/invoncify and add your translation to the desired language, if you can't find the desired language please let me know at discussion #31, I'll add it to the page and start receiving the apports
